### DISCUSSION TITLE
Empty arrays are not displayed

### DISCUSSION THREAD

**Discussion by PostIt59** [AykutSarac/jsoncrack.com#53] [General]
Hi @AykutSarac , 

I've juste discovered your tool which is amazing, it will be helpful for documentation. 

However, when an array is empty its not displayed in the graph. You'll be able to visualize this bug with the following data (addresse array is not displayed) :

```
{
  "status": "active",
  "customerInformations": {
    "firstName": "John",
    "lastName": "DOE",
    "customerNumber": "123456789"
  },
  "addresses": [],
  "_metadata": {}
}
```
Is it a missing feature or it's a choice to ignore empty arrays ?

Thanks

**Comment by AykutSarac** [comment]
Hello, empty arrays are intentionally hidden from the graph (empty objects will be hidden in future too). Do you think this should be reverted?

**Comment by PostIt59** [comment]
Hello @AykutSarac ,

In my opinion, since users want to add your graphs in documentation, it will be greater to display those empty arrays. So yes, for me, it should be reverted ;)

**Comment by nguyenab** [comment]
I think making this a toggle would be helpful. Displaying empty arrays or strings can be helpful to visualize the entire JSON structure.
